src/test/resources/features/CreateRouteVersion.feature

```diff
@@ -0,0 +1,15 @@
+Feature: Create RouteVersion
+  In order to use the app
+  As a user
+  I want to create RouteVersion and check throw errors
+
+  Background:
+    Given There is a registered user with username "username" and password "password" and email "user@domain.com"
+    Given There is a route with a title "testRoute", description "route description", type "Running" and a creationDate "2023-10-25T17:27:00Z" by user username "username"
+
+  Scenario: Create new correct RouteVersion
+    Given I login as "username" with password "password"
+    When I create a new correct RouteVersion
+    Then The response code is 201
```

Comment:
It should be checked that the route version has been actually created and that it has the expected attributes and properties in one or more additional Then steps

Reply:
Especially if the versionOf field is properly set